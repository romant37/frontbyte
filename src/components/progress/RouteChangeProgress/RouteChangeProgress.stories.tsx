import React from 'react'
import RouteChangeProgress from './RouteChangeProgress'

export default {
  title: 'UI | RouteChangeProgress',
}

export function story() {
  return <RouteChangeProgress />
}
