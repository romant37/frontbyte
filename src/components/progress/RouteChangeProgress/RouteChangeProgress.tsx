import React from 'react'
import { Wrapper } from './RouteChangeProgress.styles'

const RouteChangeProgress: React.FC = () => {
  return <Wrapper>Loading...</Wrapper>
}

export default RouteChangeProgress
