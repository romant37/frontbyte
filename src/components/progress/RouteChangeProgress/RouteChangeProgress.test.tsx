import React from 'react'
import renderer from 'react-test-renderer'
import RouteChangeProgress from './RouteChangeProgress'

test('RouteChangeProgress', () => {
  const component = renderer.create(<RouteChangeProgress />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
