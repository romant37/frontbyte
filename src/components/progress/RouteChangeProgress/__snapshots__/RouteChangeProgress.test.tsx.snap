// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouteChangeProgress 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  Loading...
</div>
`;
